### ISSUE TITLE
RemoteServiceException on multiple upload request

### ISSUE THREAD

**Issue by sanyadita** [gotev/android-upload-service#457]
Several times app threw RemoteServiceException when I start several uploads on devices under android 8+. Unfortunately, I have only Crashlytics report and don't know how to 100% reproduce it. Every time, when it appeared, me or my tester were sending image one by one: send one, start choose another.
```
Fatal Exception: android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{9cef247 u0 ai.zin.zin/net.gotev.uploadservice.UploadService}
       at android.app.ActivityThread$H.handleMessage + 1872(ActivityThread.java:1872)
       at android.os.Handler.dispatchMessage + 106(Handler.java:106)
       at android.os.Looper.loop + 214(Looper.java:214)
       at android.app.ActivityThread.main + 7050(ActivityThread.java:7050)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run + 493(RuntimeInit.java:493)
       at com.android.internal.os.ZygoteInit.main + 965(ZygoteInit.java:965)
```

**Comment by gotev** [comment]
Can you tell me on which devices manufacturers is happening and the other data got from Crashlytics?

**Comment by sanyadita** [comment]
There are reports from release version on Samsung S8 with Android 8.0.0 and debug version on Samsung S8 with Android 9. Both devices are not rooted.

**Comment by gotev** [comment]
Since it's not replicable but I'm actively developing the 4.0.0 version which will be released soon (check #450) I advise you to update as soon as I release it and then we'll see if it happens again. Closing this.

**Comment by gotev** [comment]
@sanyadita Upload Service 4.0.0 Alpha 01, 100% Kotlin with Android 10 support is here for developer preview! [Check this issue for more information](https://github.com/gotev/android-upload-service/issues/462)
